### ISSUE TITLE
Has it Save/Load Context  functionality ?

### ISSUE THREAD

**Issue by PetroGromovo** [dbeaver/dbeaver#6611]
#### System information: 
-kubuntu 18 
- workbench 6.3 

#### Connection specification:
- Database name and version

#### Describe your question:
Using DBeaver 6.1.5 inder kubuntu 18.04 I wonder does it have feature
like Save/Load Context - which is helpfull ro switch to different set of tables, save it and reopen when I need ?

2) I do not want to use all my databases on the current connection, How can I hide some databases, without deleting them?

**Comment by PetroGromovo** [comment]
3) I see that after installation DBeaver has different fonts I have installed in my OS : https://imgur.com/a/4jiBJYi
I have found “Preferences”->"Colors and Fonts ", but I failed to apply my system fonts to the app :
https://imgur.com/a/qsDNJjV
even clicking on “Use system Font” button I do not see my system fonts applied to this app.
How can I apply my system fonts? Maybe configuring some config files?
